docs/contributing.md

```diff
@@ -114,7 +114,7 @@ You will probably also want to make the `gh-pages` branch immutable (and thereby
 hidden from the default `jj log` output) by running the following in your repo:
 
 ```shell
-jj config set --repo "revset-aliases.immutable_heads()" "main@origin | gh-pages@origin"
+jj config set --repo "revset-aliases.immutable_heads()" 'remote_branches(exact:"main") | remote_branches(exact:"origin")'
```

Comment:
Should the second alternative be `gh-pages`?

Reply:
yeah, I seem to have copy pasted the wrong segment